src/explore-education-statistics-common/src/utils/number/formatPretty.ts

```diff
@@ -20,19 +20,53 @@ export default function formatPretty(
   unit?: string,
   decimalPlaces?: number,
 ): string {
+  switch (unit) {
+    case 'numberstring':
+      return String(value);
+    case '£': {
+      let formattedNumber = formatNumber(value, decimalPlaces);
```

Comment:
this came up in the linting error
to use `const` instead of `let` as this variable is never reassigned. Same goes for the rest of the _cases_ below